### DISCUSSION TITLE
Decal on parts of a robot?

### DISCUSSION THREAD

**Discussion by rohit-kumar-j** [google-deepmind/mujoco#166] [General]
I understand that Mujoco is not intensive on graphics of parts, etc... However, is there an example to put a decal on a part of a robot in Mujoco? I had seen this on the older [forum](https://roboti.us/forum/index.php?threads/is-it-possible-to-add-a-png-as-logo.3911/). But was unable to reproduce it, especially for complex `.stl` files.

![image](https://user-images.githubusercontent.com/37873142/156626351-0d2ced3c-5e80-46f3-bed8-1484db7c32bf.png)

**Comment by nimrod-gileadi** [comment]
As of version 2.1.2, MuJoCo supports OBJ meshes, which have simpler texture mapping.
If your 3D software can export OBJ and a texture, you can use it as shown in https://github.com/deepmind/mujoco/tree/main/model/mug
